app/models/form/lettings/questions/person_working_situation.rb

```diff
@@ -29,6 +29,6 @@ def initialize(id, hsh, page, person_index:)
     },
     "0" => { "value" => "Other" },
     "divider" => { "value" => true },
-    "10" => { "value" => "Tenant prefers not to say" },
```

Comment:
unrelated fix - this is not for the lead tenant so for consistency we should say person here as we do for the other extra person age/sex/relat questions

Reply:
Yeah it's not wrong per se, it's just to be consistent with other questions - it might be worth revisiting the naming to make suer they're all what we want them to be rather than just consistent with each other, I'll make a tech debt ticket for it but this is fine for now